qml/main.qml

```diff
@@ -326,17 +329,36 @@ ApplicationWindow {
                 onActivated: search.forceActiveFocus()
             }
             onTextChanged: selectFirstSearchResult()
-            Keys.onEscapePressed: {
-                search.clear()
-                deselectCredential()
-            }
+            Keys.onEscapePressed: search.clear()
             Keys.onReturnPressed: generateOrCopy()
             Keys.onEnterPressed: generateOrCopy()
             Keys.onDownPressed: arrowKeys.goDown()
             Keys.onUpPressed: arrowKeys.goUp()
+
+            // Override the copy action,
+            // since this is a TextField.
+            Keys.onPressed: {
+                if(event.matches(StandardKey.Copy)) {
+                    copyAction.trigger()
+                }
+            }
+
+            function clear() {
+                search.text = ""
+                arrowKeys.forceActiveFocus()
+                deselectCredential()
```

Comment:
Sure about clearing the selection? It's now been fixed to be stable across changing search queries; I feel like it makes more sense to keep the selection where it is (by just deleting this line).

Reply:
Hmm, really? If I'm clearing the search with the ESC key, I wouldn't want any credential to be selected. I think this pattern is pretty common in search fields.

Reply:
Hm, ok. I think I'd expect that if I type "github" into the search box and get my GitHub credential selected, it'll remain selected after I hit <kbd>Esc</kbd>. But it's no big deal, I'm fine with this as is if you prefer it that way.